app/forms/concerns/waste_exemptions_engine/can_add_or_replace_an_address.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module WasteExemptionsEngine
+  module CanAddOrReplaceAnAddress
+    extend ActiveSupport::Concern
+
+    included do
+      def add_or_replace_address(address, existing_addresses, existing_address)
```

Comment:
It'd be nice to clean this up, but given that it's been like this for ages and CodeClimate is only kicking off because the method was moved, I'm happy to leave it.